Framework/Kernel/test/BinaryStreamReaderTest.h

```diff
@@ -35,13 +35,18 @@ class BinaryStreamReaderTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(std::ios_base::beg, m_bytes.tellg());
   }
 
+  void test_Read_int16_t_Gives_Correct_Value() {
+    doReadSingleValueTest<int16_t>(6, sizeof(int16_t));
+  }
+
   void test_Read_int32_t_Gives_Correct_Value() {
-    doReadSingleValueTest<int32_t>(6, sizeof(int32_t));
+    moveStreamToPosition(10);
+    doReadSingleValueTest<int32_t>(580, sizeof(int32_t));
   }
 
   void test_Read_int64_t_Gives_Correct_Value() {
-    moveStreamToPosition(10);
-    doReadSingleValueTest<int64_t>(580, sizeof(int64_t));
+    moveStreamToPosition(42);
+    doReadSingleValueTest<int64_t>(200, sizeof(int64_t));
```

Comment:
why have these values changed?

Reply:
They changed because I called the same value earlier on and I am simply making sure that the test calls something different than other tests. It was more of a test improvement.